packages/nodes-base/nodes/Vero/UserDescription.ts

```diff
@@ -22,32 +22,32 @@ export const userOperations = [
 			{
 				name: 'Alias',
 				value: 'alias',
-				description: 'Changes a user’s identifier.',
+				description: 'Changes an user’s identifier.',
 			},
 			{
 				name: 'Unsubscribe',
 				value: 'unsubscribe',
-				description: 'Unsubscribes a single user.',
+				description: 'Unsubscribe an user.',
```

Comment:
This whole document looks very different in the docs. Can you please double check?